x/ccv/provider/types/keys.go

```diff
@@ -161,6 +161,7 @@ const (
 	ToBeOptedOutBytePrefix
 
 	// NOTE: DO NOT ADD NEW BYTE PREFIXES HERE WITHOUT ADDING THEM TO getAllKeyPrefixes() IN keys_test.go
+	ConsumerRewardPoolByteBytePrefix
```

Comment:
nit: Move this before the comment for consistency, and add a comment as explanation like for the other definitions